test/MezzioInstallerTest/HomePageResponseTest.php

```diff
@@ -218,7 +229,7 @@ public function installCasesProvider(): Generator
                     continue;
                 }
 
-                foreach ($this->intallTypes as $intallType) {
+                foreach ($this->installTypes as $intallType) {
```

Comment:
You could correct the `$intallType` typos here while you're fixing `$this->intallType`.

```suggestion
                foreach ($this->installTypes as $installType) {
```